packages/core/src/utilities/createSigmoidRGBTransferFunction.ts

```diff
@@ -0,0 +1,48 @@
+import vtkColorTransferFunction from '@kitware/vtk.js/Rendering/Core/ColorTransferFunction';
+import { VOIRange } from '../types';
+
+/**
+ * A utility that can be used to generate an RgbTransferFunction
+ *
+ * @example
+ * Setting an RGB Transfer function from the viewport:
+ * ```
+ * const sigmoidRGBTransferFunction = createSigmoidRGBTransferFunction(0, 255, { lower: 0, upper: 255} );
+ * viewport
+ *   .getActor()
+ *   .getProperty()
+ *   .setRGBTransferFunction(0, sigmoidRGBTransferFunction);
+ * ```
+ *
+ * @see {@link https://kitware.github.io/vtk-js/api/Rendering_Core_ColorTransferFunction.html|VTK.js: ColorTransferFunction}
+ * @param rgbTransferFunction
+ */
+export default function createSigmoidRGBTransferFunction(
+  windowWidth: number,
+  windowCenter: number,
+  voiRange: VOIRange
+): vtkColorTransferFunction {
+  const cfun = vtkColorTransferFunction.newInstance();
+  const sigmoid = (x: number, wc: number, ww: number) =>
+    1 / (1 + Math.exp((-4 * (x - wc)) / ww));
```

Comment:
This is not a regular sigmoid equation as I see, where is this -4 coming from? can we add some inline docs?

Reply:
https://dicom.nema.org/medical/dicom/2018b/output/chtml/part03/sect_C.11.2.html

Added a reference to dicom nema which uses the -4